apps/web/src/server/applications/pages/case/applications-case.controller.js

```diff
@@ -1,37 +1,51 @@
 import { publishCase } from '../../lib/services/case.service.js';
 
 /** @typedef {import('../../applications.types').Case} Case */
+
 /** @typedef {import('@pins/express').ValidationErrors} ValidationErrors */
 
 /**
  * View the details for a single case
  *
- * @type {import('@pins/express').RenderHandler<{selectedPageType: string, showPublishedBanner: boolean}, {}, {}, {published?: string}, {pageType?: string}>}
+ * @type {import('@pins/express').RenderHandler<{selectedPageType: string, showPublishedBanner: boolean, isFirstTimePublished: boolean}, {}, {}, {published?: string}, {pageType?: string}>}
  */
 export async function viewApplicationsCasePages({ params, query }, response) {
 	// note: case details for this case are held in response.locals.case
 	const { pageType } = params;
-	const showPublishedBanner = query.published === '';
+	const { published } = query;
+	const showPublishedBanner = published === '1' || published === '2';
```

Comment:
yes, for the banner to appear. specifically, published=1 means is the first time the case is published, published=2 means is the 2nd+ time (different wording applies)

Reply:
can't you infer if it's the first or second+ publishing by looking a the application data

Reply:
no, because once you land on this page (the project information) all the information you have is the publishedDate and there's no way to know whether you set it or just updated. So you have to pass the information from the previous page and the easiest way to do so is by parameters.

If published=1 and 2 are not good, I could write them as slightly different URLs. Such as _/project-information/published_ and _/project-information/updated_

Reply:
I'm just not sure the URL is the best place for it - how about passing it through an hidden field?